committee/committee.go

```diff
@@ -90,8 +90,12 @@ func (c *committee) Update(lastRound int16, joined []*validator.Validator) {
 	}
 
 	sort.SliceStable(oldestFirst, func(i, j int) bool {
-		return oldestFirst[i].Value.(*validator.Validator).LastSortitionHeight() <
-			oldestFirst[j].Value.(*validator.Validator).LastSortitionHeight()
+		valI, valIok := oldestFirst[i].Value.(*validator.Validator)
```

Comment:
Can't we use generic here to avoid this type conversion?

Reply:
I don't have idea now, I will search and try to make it done

Reply:
fixed in :
- #667